src/brands/peapods/lib/PeaAccountProfile.js

```diff
@@ -236,6 +258,18 @@ const PeaAccountProfile = ({
       }
 
       const content = contentRef.current;
+
+      let avatarScale = Math.min(1 - content.scrollTop / 80, 1);
+      if (avatarScale < 0.4) {
```

Comment:
Can you make this more gradual so that the size of the avatar is the same size as the one in the header at the moment the header becomes visible?